persistence/sql/migrations/sql/20221109114303000000_add_courier_send_attempts_table.up.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE courier_message_dispatches (
+  dispatch_id UUID PRIMARY KEY,
+  message_id UUID NOT NULL,
+  status TEXT NOT NULL,
+  error TEXT,
```

Comment:
If we use JSON based serialization this should be `json`

Reply:
It's not, we just dump whatever we get from the SMTP library in here as plaintext. I don't think we need it to be JSON, as SMTP doesn't communicate with JSON, afaik.

Reply:
(y)